modules/op-enabling-manual-approval-gate.adoc

```diff
@@ -0,0 +1,51 @@
+// This module is included in the following assemblies:
+// * create/using-manual-approval.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-enabling-manual-approval-gate_{context}"]
+= Enabling the manual approval gate controller
+
+To use manual approval tasks, you must first enable the manual approval gate controller.
+
+.Prerequisites
+
+* You installed the {pipelines-title} Operator in your cluster.
+* You are logged on to the cluster using the `oc` command-line utility.
+* You have administrator permissions for the `openshift-pipelines` namespace.
+
+.Procedure
+
+. Create a file named `manual-approval-gate-cr.yaml` with the following manifest for the `ManualApprovalGate` custom resource (CR):
++
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: ManualApprovalGate
+metadata:
+  name: manual-approval-gate
+spec:
+  targetNamespace: openshift-pipelines
+----
+
+. Apply the `ManualApprovalGate` CR by entering the following command:
++
+[source,terminal]
+----
+$ oc apply -f <manual-approval-gate-cr>.yaml
```

Comment:
Nit - per the example in our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference), we typically use underscores in between words for user replaced values:
```suggestion
$ oc apply -f <manual_approval_gate_cr>.yaml
```

Reply:
Fixed - this was supposed to be the actual filename